studio/src/lib/schema-helpers.ts

```diff
@@ -120,14 +121,7 @@ export const mapGraphQLType = (
     return {
       ...common,
       category: "inputs",
-      fields: Object.values(graphqlType.getFields()).map((field) => ({
-        name: field.name,
-        description: field.description || "",
-        deprecationReason: field.deprecationReason || "",
-        defaultValue: field.defaultValue,
-        type: field.type.toString(),
-        loc: field.astNode?.loc,
-      })),
+      fields: Object.values(graphqlType.getFields()).map(field => maybeParseField(field.astNode)!).filter(Boolean),
```

Comment:
Can you write this without `!`? that looks like a code smell imo

Reply:
The thing is that it would be needed either after the `maybeParseField` or after the `field.astNode`;  `astNode` is nullable

Reply:
Why does `maybeParseField` return a nullable result at all if we're going to panic-level non-null assert it here?

Reply:
Originally, it didn't include the `InputValueDefinition`, but this has changed since, I've updated the code to remove `maybeParseField` as is redundant